### DISCUSSION TITLE
Better Highlight Feature with Shortcuts and Colors

### DISCUSSION THREAD

**Discussion by tom-zane** [sumatrapdfreader/sumatrapdf#3345] [Ideas]
- Click and Drag Highlight feature
- Having different colors for highlighting with keyboard shortcuts (For Ex. Pressing G + CLICK & DRAG gives GREEN Highlight, R for red highlight or some other combination of shortcuts)
- CTRL + Z Undoes the previous highlight
- Ability to add Custom Palettes and Opacities to highlight feature

**Comment by sdtsztom** [comment]
The second point above is also what I want. Hope able to bind shortcuts with different annotation tools and colors in Advanced Options.

**Comment by dungsaga** [comment]
There's a lightweight PDF reader that support multiple highlight color. It's Sioyek (https://github.com/ahrm/sioyek)
